src/sidebar/services/test/groups-test.js

```diff
@@ -236,6 +275,25 @@ describe('groups', function() {
       });
     });
 
+    it('sets the linked group to take precedence over the group saved in local storage', () => {
```

Comment:
```suggestion
    it('sets the direct-linked group to take precedence over the group saved in local storage', () => {
```

A small suggestion to make the terminology a little more consistent with the comments in the implementation of the service.